pallets/funding/src/tests/2_evaluation.rs

```diff
@@ -0,0 +1,552 @@
+						project_id
+					),
+					Error::<TestRuntime>::NotAllowed
+				);
+			});
+
+			inst.execute(|| {
+				assert_noop!(
+					PolimecFunding::start_evaluation(
+						RuntimeOrigin::signed(issuer),
+						get_mock_jwt(issuer, InvestorType::Retail, generate_did_from_account(issuer)),
+						project_id
+					),
+					Error::<TestRuntime>::NotAllowed
+				);
+			});
+		}
+
+		#[test]
+		fn evaluation_started_already() {
+			let mut inst = MockInstantiator::new(Some(RefCell::new(new_test_ext())));
+			let issuer = ISSUER_1;
+			let project_metadata = default_project_metadata(inst.get_new_nonce(), issuer);
+
+			let project_id = inst.create_new_project(project_metadata, issuer);
+			let jwt = get_mock_jwt(issuer, InvestorType::Institutional, generate_did_from_account(issuer));
+			assert_eq!(inst.get_project_details(project_id).status, ProjectStatus::Application);
+			assert_ok!(inst.execute(|| PolimecFunding::start_evaluation(
+				RuntimeOrigin::signed(issuer),
+				jwt.clone(),
+				project_id
+			)));
+			assert_eq!(inst.get_project_details(project_id).status, ProjectStatus::EvaluationRound);
+
+			inst.execute(|| {
+				assert_noop!(
+					PolimecFunding::start_evaluation(RuntimeOrigin::signed(issuer), jwt, project_id),
+					Error::<TestRuntime>::ProjectNotInApplicationRound
+				);
+			});
+		}
+
+		#[test]
+		fn no_offchain_hash_provided() {
+			let mut inst = MockInstantiator::new(Some(RefCell::new(new_test_ext())));
+			let issuer = ISSUER_1;
+			let mut project_metadata = default_project_metadata(inst.get_new_nonce(), issuer);
+			project_metadata.offchain_information_hash = None;
+
+			let project_id = inst.create_new_project(project_metadata, issuer);
+			let jwt = get_mock_jwt(issuer, InvestorType::Institutional, generate_did_from_account(issuer));
+			assert_eq!(inst.get_project_details(project_id).status, ProjectStatus::Application);
+			inst.execute(|| {
+				assert_noop!(
+					PolimecFunding::start_evaluation(RuntimeOrigin::signed(issuer), jwt, project_id),
+					Error::<TestRuntime>::MetadataNotProvided
+				);
+			});
+		}
+
+		#[test]
+		fn different_account() {
+			let mut inst = MockInstantiator::new(Some(RefCell::new(new_test_ext())));
+			let issuer = ISSUER_1;
+			let project_metadata = default_project_metadata(inst.get_new_nonce(), issuer);
+
+			let project_id = inst.create_new_project(project_metadata, issuer);
+			let jwt = get_mock_jwt(issuer, InvestorType::Institutional, generate_did_from_account(issuer));
+			assert_eq!(inst.get_project_details(project_id).status, ProjectStatus::Application);
+			assert_ok!(inst.execute(|| PolimecFunding::start_evaluation(
+				RuntimeOrigin::signed(issuer),
+				jwt,
+				project_id
+			)));
+			assert_eq!(inst.get_project_details(project_id).status, ProjectStatus::EvaluationRound);
+
+			inst.execute(|| {
+				assert_noop!(
+					PolimecFunding::start_evaluation(
+						RuntimeOrigin::signed(issuer + 1),
```

Comment:
Small nit, can we put here ISSUER_2 instead of issuer + 1?

Reply:
Did it on purpose to make it clear we want another issuer, but sure